src/host/renderData.hpp

```diff
@@ -41,6 +41,22 @@ class RenderData final : public Microsoft::Console::Render::IRenderData
     const bool IsGridLineDrawingAllowed() noexcept override;
 
     std::vector<Microsoft::Console::Types::Viewport> GetSelectionRects() noexcept override;
+    bool IsAreaSelected() const override;
+    void ClearSelection() override;

```

Comment:
Yeah that's what I was thinking, but I'm also thinking that I'm tired of not shipping 0.3 because of accessibility so I'd rather have something work NOW and let him shuffle the functions around between 0.3 and 0.4 in a TODO.

Reply:
Created issue #1992